### ISSUE TITLE
Docs don't say what the default database user ID & password are.

### ISSUE THREAD

**Issue by Stokestack** [supabase/supabase#27940]
# Improve documentation

## Link 
https://supabase.com/docs/guides/cli/local-development

## Describe the problem
I followed all the directions in "Local development," but could not access the database because they don't say what the default user ID and password are.

Is the documentation missing?
Yes

## Describe the improvement
Tell the user what the default user ID and password are. This belongs in the section entitled, "Access your project's services."

Currently that section says:
> You can now visit your local Dashboard at http://localhost:54323/, and access the database directly with any Postgres client via postgresql://postgres:postgres@localhost:54322/postgres.

The next statement should be, "Using the default username and password: something/something."

## Additional context
![tellDefaultDBCredentials](https://github.com/user-attachments/assets/86537a91-5440-4f65-8aaa-5316f32bddd0)

**Comment by silentworks** [comment]
The connection string shown includes the username and password in it. Or are you referring to a different username/password here?

**Comment by Stokestack** [comment]
Thanks for the follow-up. I don't think it's safe to assume that the reader will know that this string contains the username and password (I didn't.).

Additionally, some DB-admin applications don't use a string like that. For example, I'm using Beekeeper Studio; its connection dialog takes a username and password as I've seen many others do.

![Screenshot 2024-07-13 at 2 50 55 PM](https://github.com/user-attachments/assets/b39b2b11-7f71-4fba-b0eb-824cbba64122)

**Comment by silentworks** [comment]
Personally I think this is an advanced feature and anyone doing this would/should know about postgres connection strings and what they contain. In reference to your screenshot here, beekeeper studios and most if not all DB admin tools have an `Import from URL` feature like in your screenshot.

**Comment by encima** [comment]
Agree with @silentworks here, I think this is OK to close as the information is there. If others experience this, please comment (or react with emojis) and we can look to include this.

**Comment by Stokestack** [comment]
Thanks for the replies, but I strongly disagree. I have used several major databases over the years, but never Postgres. I've programmed for MySQL (& MariaDB), SQL Server, SQLite, Oracle PL-SQL. It is completely reasonable to expect that someone's first exposure to Postgres will be motivated by his desire to use Supabase.

I don't understand the resistance to simply putting

_(using the default username/password of postgres/postgres)_

in the doc. It's 58 characters.
